string objectName
string detectorName
---
int64 result
